<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monsys::MonitoringView</class>
 <widget class="QMainWindow" name="monsys::MonitoringView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MonitoringView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cpu_tab">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QScrollArea" name="cpu_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>758</width>
             <height>511</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QChartView" name="cpu_load_view" native="true"/>
            </item>
            <item>
             <widget class="QChartView" name="cpu_process_view" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="memory_tab">
       <attribute name="title">
        <string>Memory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="memory_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="memory_scroll_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>758</width>
             <height>511</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QChartView" name="ram_total_view" native="true"/>
            </item>
            <item>
             <widget class="QChartView" name="ram_view" native="true"/>
            </item>
            <item>
             <widget class="QChartView" name="hard_volume_view" native="true"/>
            </item>
            <item>
             <widget class="QChartView" name="hard_ops_view" native="true"/>
            </item>
            <item>
             <widget class="QChartView" name="hard_throughput_view" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="network_tab">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QScrollArea" name="network_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="network_scroll_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>758</width>
             <height>511</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QChartView" name="url_available_view" native="true"/>
            </item>
            <item>
             <widget class="QChartView" name="inet_throughput_view" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
